### ISSUE TITLE
Save log files with a proper .log ending

### ISSUE THREAD

**Issue by nestire** [Nitrokey/pynitrokey#317]
At the moment log files are saved like that
`nitropy.log.yndzwd59`
This prevents fast upload to pages like github since they check for the _.log_ ending and confuses some editors.

please move the random  string in front of the file ending, to prevent the renaming step.

**Comment by robin-nitrokey** [comment]
Good point, we should just use something like `tempfile.NamedTemporaryFile(prefix="nitropy.", suffix=".log")`.

https://github.com/Nitrokey/pynitrokey/blob/d18dcd1ad013b47484f01ee3a86019bc2fee639e/pynitrokey/confconsts.py#L40
